internal/project/project.go

```diff
@@ -192,6 +213,11 @@ func (p *Project) GetCompilerOptions() *core.CompilerOptions {
 
 // GetSourceFile implements compiler.CompilerHost.
 func (p *Project) GetSourceFile(fileName string, path tspath.Path, languageVersion core.ScriptTarget) *ast.SourceFile {
+	if p.getCachedSourceFile != nil {
```

Comment:
This is just a draft, but I would think we'd want to actually do this down where we do the parse call, since this will prevent us from testing the doc registry logic.

Reply:
Working on a fix.